integration/eth2network/eth2_configs.go

```diff
@@ -40,7 +44,7 @@ ALTAIR_FORK_VERSION: 0x20000090
 # Merge
 BELLATRIX_FORK_EPOCH: 4
 BELLATRIX_FORK_VERSION: 0x20000091
-TERMINAL_TOTAL_DIFFICULTY: 2
+TERMINAL_TOTAL_DIFFICULTY: 10
```

Comment:
Why the change?

Reply:
It was reaching the difficulty too fast and seem'd to get the network in a stale state.